.coveragerc

```diff
@@ -156,7 +156,10 @@ omit =
     homeassistant/components/ebox/sensor.py
     homeassistant/components/ebusd/*
     homeassistant/components/ecoal_boiler/*
-    homeassistant/components/ecobee/*
+    homeassistant/components/ecobee/binary_sensor.py
```

Comment:
Please exclude the init module too.

Reply:
Just checked codecov and realized I forgot to add tests for the options flow... will add those tonight.